Patch-Source: https://github.com/llvm/llvm-project/commit/60e32a1f34e9ea60155a98bbe6ee5ec2a383efa3
From 60e32a1f34e9ea60155a98bbe6ee5ec2a383efa3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 13 Jun 2021 12:32:07 -0700
Subject: [PATCH] [fuzzer] Fix build on musl

cstddef is needed for size_t definition.
(Multiple headers can provide size_t but none of them exists.)

Reviewed By: MaskRay

Differential Revision: https://reviews.llvm.org/D96213
---
 compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp b/compiler-rt/lib/fuzzer/FuzzerInterceptors.cpp
index b87798603fda5..d5b0a42fd3fb4 100644
--- a/lib/fuzzer/FuzzerInterceptors.cpp
+++ b/lib/fuzzer/FuzzerInterceptors.cpp
@@ -25,6 +25,7 @@
   }
 
 #include <cassert>
+#include <cstddef> // for size_t
 #include <cstdint>
 #include <dlfcn.h> // for dlsym()
 
